jekyll/_cci2/using-orbs.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: classic-docs
+title: "Using Orbs"
+short-title: "Using Orbs"
+description: "Starting point for CircleCI Orbs"
+categories: [getting-started]
+order: 1
+---
+
+This document provides a basic example of using an inline Orb and elements of Orbs followed by a conceptual overview of Orbs design.
+
+* TOC
+{:toc}
+
+## Introduction
+
+Orbs are package systems (similar to Ruby Gems) that you can use to quickly and easily get up and running using the CircleCI platform. 
+
+To use an existing orb inline in your `config.yml` file, invoke it with the `orbs` key. The following example invokes the `hello-build` orb in the `circleci` namespace.
+
+```
+version: 2.1
+
+orbs:
+    hello: circleci/hello-build@volatile
+
+workflows:
+    "Hello Workflow":
+        jobs:
+          - hello/hello-build
+```
+
+Orbs encourage abstraction of a tight scope with a usable and flexible interface.
+
+Orbs consist of the following elements:
+
+* Commands
+* Jobs
+* Executors
+
+### Commands
+{:.no_toc}
+
+Commands are reusable sets of steps that you can invoke with specific parameters within an existing job (e.g. `checkout` and `run`). For example, if you want to invoke the command `sayhello`, you would pass the parameter `to` as follows:
+
+```
+jobs
+  myjob:
+    docker:
+      - image: "circleci/node:9.6.1"
+    steps:
+      - sayhello:
+          to: "Lev"
+```
+
+### Jobs
+{:.no_toc}
+
+Jobs are sets of steps and the environments that the steps should be executed within the environment. Jobs are defined in your build configuration or in an orb and enable you to define a job name in a map under the `jobs` key in a configuration, or in an external orb's configuration.
+
+<aside class="notice">
+You must invoke jobs in the workflow stanza of `config.yml`, making sure to pass any necessary parameters as subkeys to the job.
+</aside>
+
+### Executors
+{:.no_toc}
+
+Executors define the environment in which the steps of a job will be run. When you declare a `job` in CircleCI configuration, you define the type of environment (e.g. `docker`, `machine`, `macos`, etc.) to run in, in addition to any other parameters of that environment, such as:
+
+* environment variables to populate
+* which shell to use
+* what size `resource_class` to use
+
+When you declare an executor in a configuration outside of `jobs`, you can use these declarations for all jobs in the scope of that declaration, enabling you to reuse a single executor definition across multiple jobs.
+
+Each executor definition includes the subset of the children keys of a `job` declaration related to setting the environment for a job to execute. This means it does not include `steps`. The subset can be one or more of the following keys:
+
+* `docker`, `machine`, or `macos`
+* `environment`
+* `working_directory`
+* `shell`
+* `resource_class`
```

Comment:
TODO: I have vague memory we decided to not allow resource_class in orbs, so let's check with @johnswanson to be sure.

Reply:
@ndintenfass it looks like we allow it, though possibly we shouldn't?

Reply:
If we allow it we can document that. If we change that, we'll change the docs later.

Reply:
leaving this for now